package.json

```diff
@@ -1,17 +1,28 @@
 {
-  "name": "databraid-react-starter",
+  "name": "transit-spa",
   "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "enzyme": "^2.9.1",
```

Comment:
Actually don't worry about squashing your commits for now. Please just fix the other things. thanks.